daemon/oci_windows.go

```diff
@@ -214,6 +214,33 @@ func (daemon *Daemon) createSpec(ctx context.Context, c *container.Container) (*
 	return &s, nil
 }
 
+// getLayerFolders returns the layer folders from an image RootFS
+func (daemon *Daemon) getLayerFolders(img *image.Image, rwLayer layer.RWLayer) ([]string, error) {
```

Comment:
We could even remove the function from the `Daemon` and have it take the layer store as a parameter.